cloudferrylib/os/network/neutron.py

```diff
@@ -857,10 +916,23 @@ def upload_subnets(self, networks, subnets):
                 continue
             tenant_id = \
                 self.identity_client.get_tenant_id_by_name(snet['tenant_name'])
+            if not tenant_id:
```

Comment:
that's why we have this check. if method    self.identity_client.get_tenant_id_by_name(snet['tenant_name'])    doesn't return valid data - we continue